app/templates/views/broadcast/partials/area-map-postcodes.html

```diff
@@ -0,0 +1,142 @@
+<script src="{{ asset_url('javascripts/leaflet.js') }}"></script>
+<script>
+
+  var polygons = []
```

Comment:
I don't know about the rest of the code but modern javascript tends to use `let` and `const` rather than `var`.  Otherwise you end up creating global variables a lot.

Reply:
I've removed all var references and replaced with appropriate const or let.